investments/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+from .models import Subscription
+
+
+class SubscriptionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Subscription
+        fields = ["id", "investment_share", "created_at", "updated_at"]
+        read_only_fields = ["id", "created_at", "updated_at"]
```

Comment:
As noted in the model review: if `Subscription` is meant to be linked to a `Project`, you’ll likely want to include that in the serializer:
```
fields = ["id", "investment_share", "project", "created_at", "updated_at"]
```